q2_dada2/plugin_setup.py

```diff
@@ -189,6 +191,8 @@
                     'read is then shorter than `trunc_len_f` or `trunc_len_r` '
                     '(depending on the direction of the read) it is '
                     'discarded.'),
+        'min_overlap': ('The minimum number of matched overlapping reads '
+                        'needed to merge two sequences.'),
```

Comment:
I think we can get away with copying the dada2 docstring for this one, since we are just passing this param straight through:

```suggestion
        'min_overlap': ('The minimum length of the overlap required for '
                        'merging the forward and reverse reads.'),
```